src/editor/codemirror/language-server/documentation.ts

```diff
@@ -30,3 +26,17 @@ export const renderDocumentation = (
   div.appendChild(new Text(fallbackContent));
   return div;
 };
+
+export interface SanitisedHtml {
+  __html: string;
+}
+
+export const renderMarkdown = (markdown: string): SanitisedHtml => {
+  return {
+    __html: DOMPurify.sanitize(
+      render(markdown, {
+        gfm: true,
+      })
+    ),
+  };
+};
```

Comment:
Now we've exposed this directly it should have coverage to check for sanitisation.